lang/go/go.py

```diff
@@ -134,7 +134,7 @@ def code_insert_is_not_null():
         actions.insert(" != nil")
 
     def code_state_for():
-        actions.user.insert_snippet_by_name("whileLoopStatement")
+        actions.user.insert_snippet_by_name("forLoopStatement")
```

Comment:
All of these are now defined on the module so you can remove them for the specific languages